test/api/repositories/authentication_repository_test.dart

```diff
@@ -0,0 +1,57 @@
+import 'package:flutter_config/flutter_config.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+import 'package:survey_flutter/repositories/authentication_repository.dart';
+import 'package:survey_flutter/api/exception/network_exceptions.dart';
+import 'package:survey_flutter/model/response/login_response.dart';
+
+import '../../mocks/generate_mocks.mocks.dart';
+
+void main() {
+  group('AuthenticationRepositoryTest', () {
+    late MockAuthenticationApiService mockAuthApiService;
+    late AuthenticationRepositoryImpl authRepository;
+
+    const email = "email";
+    const password = "password";
+
+    setUpAll(() {
+      FlutterConfig.loadValueForTesting({
+        'CLIENT_ID': 'CLIENT_ID',
+        'CLIENT_SECRET': 'CLIENT_SECRET',
+      });
+    });
+
+    setUp(() {
+      mockAuthApiService = MockAuthenticationApiService();
+      authRepository = AuthenticationRepositoryImpl(mockAuthApiService);
+    });
+
+    test('When login successfully, it returns correct model', () async {
+      final loginResponse = LoginResponse(
+        id: "",
+        accessToken: "",
+        tokenType: "",
+        expiresIn: 0,
+        refreshToken: "",
+        createdAt: 0,
+      );
```

Comment:
This is optional 🙏 Is it possible to put this instance creation as a static method for `LoginResponse` so we can use it in other locations 🤔 
Something like:
```dart
final loginResponse = LoginResponse.dummy()
```

Reply:
Good suggestion 👍 , I updated all at https://github.com/nimblehq/flutter-ic-khanh-thieu/pull/51/commits/dff6a2549abeb02e72cf50686bbfbdc14ffaba4f 🙏